internal/step/plugin/provider.go

```diff
@@ -338,6 +368,60 @@ func (r *runnableStep) StartedSchema() *schema.StepOutputSchema {
 	)
 }
 
+func (r *runnableStep) EnabledOutputSchema() *schema.StepOutputSchema {
+	return schema.NewStepOutputSchema(
+		schema.NewScopeSchema(
+			schema.NewObjectSchema(
+				"EnabledOutput",
+				map[string]*schema.PropertySchema{
+					"enabled": schema.NewPropertySchema(
+						schema.NewBoolSchema(),
+						schema.NewDisplayValue(
+							schema.PointerTo("enabled"),
+							schema.PointerTo("Whether the step was enabled"),
+							nil),
+						true,
+						nil,
+						nil,
+						nil,
+						nil,
+						nil,
+					),
+				},
+			),
+		),
+		nil,
+		false,
+	)
+}
+
+func (r *runnableStep) DisabledOutputSchema() *schema.StepOutputSchema {
+	return schema.NewStepOutputSchema(
+		schema.NewScopeSchema(
+			schema.NewObjectSchema(
+				"DisabledMessageOutput",
+				map[string]*schema.PropertySchema{
+					"message": schema.NewPropertySchema(
+						schema.NewStringSchema(nil, nil, nil),
+						schema.NewDisplayValue(
+							schema.PointerTo("message"),
+							schema.PointerTo("A human readable message stating that the step was disabled."),
```

Comment:
This feels very ... boilerplate. Is this really what you intended?

Reply:
It's just a way for users to have a readable output for the disabled stage.

Reply:
I mean, did you really intend to say "A human readable message"???

Reply:
This is a very different pattern from the other places you've added a `schema.NewDisplayValue`, and it looks odd. E.g., compared to "Whether the step was enabled" above or "Used to set whether the step is enabled" below.

Reply:
The main hint that was made with that sentence is that it's not intended for further processing. It's just a message for the users. What would you rather it say?

Reply:
It doesn't dictate the contents, but it should correlate well with the contents.